src/Microsoft.AspNet.Mvc.Razor.Host/MvcRazorHost.cs

```diff
@@ -95,7 +95,7 @@ internal MvcRazorHost(IChunkTreeCache chunkTreeCache, RazorPathNormalizer pathNo
                     ScopeManagerBeginMethodName = nameof(TagHelperScopeManager.Begin),
                     ScopeManagerEndMethodName = nameof(TagHelperScopeManager.End),
 
-                    TagHelperContentTypeName = nameof(TagHelperContent),
+                    TagHelperContentTypeName = typeof(TagHelperContent).FullName,
```

Comment:
Needs to say `global::Microsoft.Foo.Bar` to avoid conflicts with sub-namespaces named `Microsoft`.